src/EditorFeatures/CSharpTest/ChangeSignature/ReorderParametersTests.cs

```diff
@@ -33,6 +33,70 @@ public void Goo(string y, int x)
             await TestChangeSignatureViaCommandAsync(LanguageNames.CSharp, markup, updatedSignature: permutation, expectedUpdatedInvocationDocumentCode: updatedCode);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task ReorderLocalFunctionParameters()
```

Comment:
would be nice to move this below ReorderMethodParametersAndArguments and call it ReorderLocalFunctionParametersAndArguments for consistency